content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -12,7 +12,7 @@ weight: 60
 Application logs can help you understand what is happening inside your application. The logs are particularly useful for debugging problems and monitoring cluster activity. Most modern applications have some kind of logging mechanism. Likewise, container engines are designed to support logging. The easiest and most adopted logging method for containerized applications is writing to standard output and standard error streams.
 
 However, the native functionality provided by a container engine or runtime is usually not enough for a complete logging solution.
-For example, you may want access your application's logs if a container crashes; a pod gets evicted; or a node dies.
+For example, you may want to access your application's logs if a container crashes; a pod gets evicted; or a node dies.
```

Comment:
@bradtopol  I agree with these edits (lines 15 and 144).  Looks good to me.  
( ꈍᴗꈍ)

Reply:
/cc @bradtopol